storybookjs/storybook
# Handle a11y for components with delayed rendering

Issue: #2537 

## What I did

Implemented support in a11y-addon for delayed rendering of components by adding a "re-run tests" button within the panel.

I also corrected the contrast of the addon's tabs (to 7:1) and added the number of violations and passes to each tab (because if you re-run, the number may change, but you wouldn't know until you navigated to the tab).

## How to test

There's a story called "Accessibility" in cra-kitchen-sink that demonstrates a11y-addon with a regular button and with a button that doesn't render until 1s after the preview loads. In order to verify that it works, click the "re-run tests" button. (Ideally, we would be able to verify that results are displayed in the tab.)
